infrastructure/docker/php7.2-fpm/Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM php:7.2-fpm-alpine
+LABEL maintainer="ITK Dev <itkdev@mkb.aarhus.dk>"
+
+ENV PHP_OPCACHE_VALIDATE_TIMESTAMPS="0" \
+    PHP_OPCACHE_MAX_ACCELERATED_FILES="20000" \
+    PHP_OPCACHE_MEMORY_CONSUMPTION="192" \
+    PHP_OPCACHE_MAX_WASTED_PERCENTAGE="10" \
+    PHP_MAX_EXECUTION_TIME="30" \
+    PHP_MEMORY_LIMIT="128M" \
+    PHP_POST_MAX_SIZE="8M" \
+    PHP_UPLOAD_MAX_FILESIZE="2M"
```

Comment:
For the upload service this is a bit small but can be changed later.

Reply:
Yes, this is the generic image and not done yet. This is made as env variables, so they can be overriden for the app images :-)